UNITTESTS/drivers/PwmOut/test_pwmout.cpp

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2019 Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "drivers/PwmOut.h"
+
+using namespace mbed;
+
+class MbedPowerMgmtInterface {
+public:
+    virtual ~MbedPowerMgmtInterface() {}
+    virtual void sleep_manager_unlock_deep_sleep_internal() = 0;
+    virtual void sleep_manager_lock_deep_sleep_internal() = 0;
+};
+
+
+class MockMbedPowerMgmt : public MbedPowerMgmtInterface {
+public:
+    virtual ~MockMbedPowerMgmt() {}
+    MOCK_METHOD0(sleep_manager_unlock_deep_sleep_internal, void());
+    MOCK_METHOD0(sleep_manager_lock_deep_sleep_internal, void());
+
+    static MockMbedPowerMgmt *get_instance(bool create_instance)
+    {
+        if (instance == nullptr && create_instance) {
+            instance = new MockMbedPowerMgmt();
+        }
+
+        if (create_instance) {
+            ++ref_counter;
+        }
+
+        return instance;
+    }
+
+    static void release_instance()
+    {
+        --ref_counter;
+
+        if ((instance != nullptr) && (ref_counter == 0)) {
+            delete instance;
+            instance = nullptr;
+        }
+    }
+
+    MockMbedPowerMgmt(MockMbedPowerMgmt const&) = delete;
+    void operator=(MockMbedPowerMgmt const&) = delete;
+private:
+    MockMbedPowerMgmt() {}
+
+    static MockMbedPowerMgmt *instance;
+    static int ref_counter;
```

Comment:
Add an underscore to private members to follow convention.

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/f6fb26510d8ca6ff3e617a531b394a41264e66e3..ac77d3a288c6d9816bcb1601f0ef7db30e30813c).